docs: update community stappenplan (#1809)
nl-design-system/documentatie: docs/handboek/component-bijdragen/community-stappenplan/voor-kernteam.mdx

Door met [design tokens](/handboek/design-tokens/) te werken zorgen we ervoor dat meerdere organisaties de component van een eigen huisstijl kunnen voorzien. Minimaal zouden er design tokens beschikbaar moeten zijn om kleur en typografie beslissingen door te voeren.

Check of de implementatie van de gekozen organisatie hieraan voldoet. Dit doe je door de component te bekijken via de 'Storybook URL (CSS)' die in het Communitybord is ingevuld.

**Tip!** Om te controleren of er design tokens zijn toegepast kun je gebruik maken van de 'Inspect' functionaliteit van je browser. Wanneer je in de CSS verwijzingen ziet naar CSS variabelen zoals bijvoorbeeld `--denhaag-button-primary-action-background-color` of `--ams-button-font-family` kun je er vanuit gaan dat er design tokens zijn gebruikt.

> **Tip!** Soms heeft een component geen specifieke design tokens nodig. Denk bijvoorbeeld aan een design tokens voor font-family bij de componenten Mark of Icon. Gebruik in dit soort gevallen je gezonde verstand.

Reviewer: ```suggestion
**Tip!** Soms heeft een component geen specifieke design tokens nodig. Denk bijvoorbeeld aan design tokens voor font-family bij componenten zoals Mark of Icon. Gebruik in dit soort gevallen je gezonde verstand.
```

Om componenten van elkaar te kunnen gebruiken is het toevoegen van een prefix zoals bijvoorbeeld `utrecht-` of `ams-` verplicht.

> Check of de prefix van de gekozen organisatie is toegevoegd aan onderstaande onderdelen:

Reviewer: ```suggestion
Check of de prefix van de gekozen organisatie is toegevoegd aan de onderstaande onderdelen:
```

Doel: De component wordt beter vindbaar voor andere organisaties en kan makkelijk worden voorzien van een eigen thema.

> Als kernteam maken we de component beschikbaar in de ['Themes' repository](https://github.com/nl-design-system/themes), waardoor deze terug te vinden is in de [Storybook met alle NL Design System thema's](https://nl-design-system.github.io/themes/). Andere organisaties kunnen de component vervolgens gaan gebruiken.

Reviewer: ```suggestion
Als kernteam maken we de component beschikbaar in de ['Themes' repository](https://github.com/nl-design-system/themes), zodat deze terug te vinden is in de [Storybook met alle NL Design System thema's](https://nl-design-system.github.io/themes/). Andere organisaties kunnen de component vervolgens gebruiken.
```

Doel: De component wordt beschikbaar in de [NL Design System Figma bibliotheek](https://www.figma.com/design/shhwGcqPLi2CapK0P1zz8O/NLDS---Voorbeeld---Bibliotheek?node-id=450-4121&t=AKzhgDPYAKNI1gaT-1). Designers van andere organisaties kunnen de component overnemen en gaan gebruiken.

> - Als kernteam nemen we de component over uit de Figma omgeving van de gekozen organisatie. Inclusief bijbehorende design tokens. We plaatsen de component in de NL Design System Figma bibliotheek. Daarbij wordt de component voorzien van het Voorbeeld thema.

Reviewer: ```suggestion
- Als kernteam nemen we de component over uit de Figma omgeving van de gekozen organisatie, inclusief de bijbehorende design tokens. We plaatsen de component in de NL Design System Figma bibliotheek en voorzien deze van het Voorbeeld thema.
```

- Heeft de gekozen organisatie geen Figma component? Dan ontwikkelen we deze als kernteam zelf.

Organisaties die de NL Design System Figma bibliotheek vanaf dat moment dupliceren, krijgen de component direct meegeleverd. Organisaties die de NL Design System Figma bibliotheek al hadden gedupliceerd, worden tijdens de volgende 'Bieb sync video' geïnformeerd over hoe ze dit component ook kunnen overnemen.

> - Kopieer de URL van de component pagina door in Figma met rechtermuisknop op de component in de sidebar te klikken (Copy link to page).

Reviewer: ```suggestion
- Kopieer de URL van de componentpagina door in Figma met de rechtermuisknop op de component in de sidebar te klikken en 'Copy link to page' te selecteren.
```

- Plak de URL in de kolom 'Figma URL' van het [Community projectbord](https://github.com/orgs/nl-design-system/projects/29/views/1).
- Had de gekozen organisatie zelf geen Figma component? Plak de URL dan óók in het projectbord voor deze organisatie.

**🚩 Checkpoint**: Figma URL

## Beschikbaar in de NL Design System Component assessment (Figma)

Doel: Organisaties worden bij een onboarding snel naar de juiste component verwezen.

Het kernteam voegt een 'Component sticker' toe aan het Figma bestand '[NL Design System Component assessment](https://www.figma.com/design/NLbNbckFo9manEihRFeJmR/NL-Design-System---Component-Assessment?m=auto&t=J0gWvRWoyCJhSqot-6)'. Deze sticker linkt naar de documentatiepagina van de component op nldesignsystem.nl.

Door deze 'Component stickers' op screenshots te plakken, kunnen we geïnteresseerde organisaties inzicht bieden welke componenten beschikbaar zijn om hun website of applicatie mee op te bouwen.

**🚩 Checkpoint**: Component assessment

## Status bijgewerkt naar Community

Doel: iedereen kan zien dat de component nu richting Candidate wil.

> - Verander het 'Community' label van het backlog issue naar 'Candidate'.

Reviewer: ```suggestion
- Verander het label 'Community' van het backlog issue naar 'Candidate'.
```

> - Verander het label 'Community' van het backlog issue naar 'Candidate'.

Reviewer: @jeffreylauwers ik ben het stappenplan aan het bekijken en het viel me op dat hier, evenals op de volgende regel, ‘Candidate’ staat in plaats van ‘Community’. Aangezien we met de labels de huidige status aangeven, vraag ik me af waarom hier is gekozen voor ‘Candidate’.

Reviewer 2: Goeie zeg! Heb ik net nog even aangepast!

> - Verander het 'Community' label van de GitHub Discussion naar 'Candidate'.

Reviewer: ```suggestion
- Verander het label 'Community' van de GitHub Discussion naar 'Candidate'.
```

> - Filter het [Candidate bord](https://github.com/orgs/nl-design-system/projects/32) op de component door op `{naam-component}` te zoeken.

Reviewer: ```suggestion
- Filter het [Candidate bord](https://github.com/orgs/nl-design-system/projects/32) op de component door te zoeken op `{naam-component}`.
```

[{naam-component} op het Candidate bord]({url-candidate-bord})
```

**🚩 Checkpoint**: Status

> ## Over promotie

Reviewer: Waarom noemen we dit 'over' promotie?

Author: Omdat de Heading 'Gebruik van component uit de community gepromoot' ná de Component Progress stap hebben gezet en we toch iets van een Heading nodig hebben.

> Zet het checkpoint 'Promotie' op 'Done' vóór de promotie heeft plaats gevonden. Dit doen we omdat de de status 'Community' alleen zichtbaar is nadat alle checkpoint zijn behaald.

Reviewer: ```suggestion
Zet het checkpoint 'Promotie' op 'Done' vóór de promotie heeft plaatsgevonden. Dit doen we omdat de status 'Community' pas zichtbaar wordt nadat alle checkpoint zijn behaald.
```

**🚩 Checkpoint**: Promotie

## Informatie van de component bijgewerkt op nldesignsystem.nl

Doel: Op nldesignsystem.nl wordt de meest recente informatie van de component beschikbaar.

- Open de [documentatie repository](https://github.com/nl-design-system/documentatie) in Visual Studio Code.

> - Maak een nieuwe branch aan gebruik hiervoor de volgende naam:

Reviewer: ```suggestion
- Maak een nieuwe branch aan en gebruik de volgende naam:
```

- Wacht tot [component-progress op npm](https://www.npmjs.com/package/@nl-design-system/component-progress) is bijgewerkt. Je kunt dit controleren door te kijken naar de datum bij `Last publish`.
- Update `@nl-design-system/component-progress` in de documentatie repository. Dat doe je door lokaal in de branch `pnpm install @nl-design-system/component-progress@latest --save-dev -w` te draaien. Dit update de devDependency en de lockfile.
- Maak een commit en gebruik hierbij het volgende bericht:

**Commit Message**

```
build: update component progress {versienummer}
```

- Publiceer je branch naar GitHub om een pull request (PR) aan te maken.
- Ga naar GitHub en gebruik de volgende opzet voor je PR:

**Title**

```
build: update component progress {versienummer}
```

**Description**

```
This PR will update the component progress to version {versienummer}.
```

- Klik op 'Create Pull Request'
- Geef een developer uit het kernteam via Slack een seintje dat de PR klaar staat.

**🚩 Checkpoint**: nldesignsystem.nl

## Gebruik van component uit de community gepromoot

> Doel: Credits geven aan mensen die hebben bijgedragen vanuit de community. De rest van community informeren en enthousiasmeren om ook naar Community, Candidate en Hall of Fame toe te werken.

Reviewer: ```suggestion
Doel: Credits geven aan mensen die hebben bijgedragen vanuit de community. De rest van community informeren en enthousiasmeren om ook naar 'Community', 'Candidate' en 'Hall of Fame' statussen toe te werken.
```

Reviewer: We gebruiken soms wel en soms geen aanhalingstekens bij de statussen. Laten we een keuze maken of we dat wel of niet doen.

Author: Ja lastig. Ik hanteer normaal gesproken altijd 1 keer aanhalingstekens als de 'term' voor het eerst wordt geïntroduceerd. En daarna niet meer. Echt iets om een ei over te leggen in de schrijfwijzer.

Reviewer 2: Ik zeg we beginnen met een hoofdletter dus is het een eigen naam dus hoeft het geen aanhalingstekens te hebben?

> - Voeg het '📣 Promotion' label toe aan het backlog issue.

Reviewer: ```suggestion
- Voeg het label '📣 Promotion' toe aan het backlog issue.
```

- Selecteer bij ‘Projects’ het project 'Communicatie & PR'.

### Communityleden selecteren voor credits

> Het kernteam selecteert communityleden om credits aan uit te delen. Denk bijvoorbeeld aan de maintainer van Storybook en de betrokken designer. Dat kan door activiteit in de Discussion te zien of naar commits te kijken in de code van het community component.

Reviewer: ```suggestion
Het kernteam selecteert communityleden om credits toe te kennen. Dit kan bijvoorbeeld de maintainer van Storybook zijn of de betrokken designer. Activiteit in de Discussion of commits in de code van het community component kunnen hierbij als basis dienen.
```

### Slack

Plaats dit bericht in #nl-design-system, vul aan waar nodig.

```md
✨ {naam-component} is nu Community ✨

> Met dank aan {persoon/personen} van {organisatie} is {naam-component} beschikbaar in de Community!

Reviewer: ```suggestion
Met dank aan {persoon/personen} van {organisatie} is {naam-component} nu beschikbaar in de Community!
```

[Bekijk {naam-component}]({url-nldesignsystem.nl}): {beschrijving-component}

Help je mee {naam-component} naar Candidate te brengen? Dat doe je bijvoorbeeld door ons in de [GitHub Discussion voor de {naam-component} component]({url-github-discussion}) te laten weten waar je {naam-component} component gebruikt.
```

### Heartbeat, Nieuwsbrief en LinkedIn

> Vanuit het kernteam benaderen we de organisatie die de component heeft ontwikkeld. We vragen of zij de component tijdens een Heartbeat willen presenteren. Mocht dit niet mogelijk zijn doen wij dit als kernteam zelf.

Reviewer: ```suggestion
Vanuit het kernteam benaderen we de organisatie die de component heeft ontwikkeld. We vragen of zij de component tijdens een Heartbeat willen presenteren. Als dit niet mogelijk is, nemen wij als kernteam deze taak op ons.
```

Daarnaast plaatsen we een bericht in de nieuwsbrief en op LinkedIn.

Promoties hoeven niet altijd dezelfde vorm aan te houden. Gebruik onderstaande punten als voorzet:

- Naam van de component.
- Beschrijving van de component.
- URL van de component op nldesignsystem.nl
- URL van de GitHub Discussion.

> - Credits: persoon/personen van organisatie, eventueel in combinatie met een kort interview.

Reviewer: ```suggestion
- Credits: persoon/personen van organisatie, eventueel gecombineerd met een kort interview.
```

> Promoties kunnen gecombineerd worden als er meerdere componenten van status veranderen.

Reviewer: ```suggestion
Promoties kunnen ook gecombineerd worden wanneer meerdere componenten van status veranderen.
```

## 🏁 Finish

> Zo wat een werk! Je hebt alle stappen genomen en zo alle checkpoints behaald die nodig zijn voor de ['Community' status](https://github.com/orgs/nl-design-system/projects/29/views/1) van het [Estafettemodel](https://www.nldesignsystem.nl/componenten/definition-of-done). De component gaat nu door voor de 'Candidate' status.

Reviewer: Hier plaatsen we bijv. weer aanhalingstekens bij Community en Candidate.

Author: Ja en dat komt dan weer omdat dit als 'los' bericht in Slack wordt gedeeld.
